WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -1062,7 +1062,6 @@ e_hunnic_empire = {
 	allow = {
 		culture = hunnic
 		prestige = 5000
-		religion_group = pagan_group
```

Comment:
@loup99 

No, no. ANY Hun should be able to reform the Hunnic Empire. The idea that someone loses their identity then they find Jesus is silly. Christian Mongols can still hold the Nerge and steppe nomads don't stop riding when they find Jesus, so why should the Huns?

I am willing, though, to exclude Merchant Republics.